Packs/IdentityRecordedFuture/Layouts/layoutscontainer-Recorded_Future_Playbook_Alert_Identity_Exposure.json

```diff
@@ -0,0 +1,869 @@
+                                "scriptId": "DisableUserWrapper",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {},
+                                "buttonClass": "secondary",
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "d5edf6d0-0713-11ef-b715-15598fdfcb5d",
+                                "index": 1,
+                                "listId": "caseinfoid-a6089c90-0713-11ef-b715-15598fdfcb5d",
+                                "name": "Clear User's Sessions - Okta",
+                                "scriptId": "Okta v2|||okta-clear-user-sessions",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {},
+                                "buttonClass": "secondary",
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "d8e2ccd0-0713-11ef-b715-15598fdfcb5d",
+                                "index": 2,
+                                "listId": "caseinfoid-a6089c90-0713-11ef-b715-15598fdfcb5d",
+                                "name": "Revoke User's Sessions - Azure",
+                                "scriptId": "Microsoft Graph User|||msgraph-user-session-revoke",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {},
+                                "buttonClass": "secondary",
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "d802a600-0713-11ef-b715-15598fdfcb5d",
+                                "index": 3,
+                                "listId": "caseinfoid-a6089c90-0713-11ef-b715-15598fdfcb5d",
+                                "name": "Reset Password - Active Directory",
+                                "scriptId": "Active Directory Query v2|||ad-set-new-password",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Initiate Response Tasks",
```

Comment:
Same with colors can be here if you want :)

Reply:
reviewed